_config.yml

```diff
@@ -76,166 +51,223 @@ optional_components:
+      source_release:
+          name: "Apache Flink 1.8.0"
+          id: "180-download-source"
+          url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.8.0/flink-1.8.0-src.tgz"
+          asc_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-src.tgz.asc"
+          sha512_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-src.tgz.sha512"
+      optional_components:
+        -
+          name: "Pre-bundled Hadoop 2.4.1"
+          category: "Pre-bundled Hadoop"
+          scala_dependent: false
+          id: 180-bundled-hadoop-241
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.4.1-1.8.0/flink-shaded-hadoop2-uber-2.4.1-1.8.0.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.4.1-1.8.0/flink-shaded-hadoop2-uber-2.4.1-1.8.0.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.4.1-1.8.0/flink-shaded-hadoop2-uber-2.4.1-1.8.0.jar.sha1
+        -
+          name: "Pre-bundled Hadoop 2.6.5"
+          category: "Pre-bundled Hadoop"
+          scala_dependent: false
+          id: 180-bundled-hadoop-265
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.6.5-1.8.0/flink-shaded-hadoop2-uber-2.6.5-1.8.0.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.6.5-1.8.0/flink-shaded-hadoop2-uber-2.6.5-1.8.0.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.6.5-1.8.0/flink-shaded-hadoop2-uber-2.6.5-1.8.0.jar.sha1
+        -
+          name: "Pre-bundled Hadoop 2.7.5"
+          category: "Pre-bundled Hadoop"
+          scala_dependent: false
+          id: 180-bundled-hadoop-275
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.7.5-1.8.0/flink-shaded-hadoop2-uber-2.7.5-1.8.0.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.7.5-1.8.0/flink-shaded-hadoop2-uber-2.7.5-1.8.0.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.7.5-1.8.0/flink-shaded-hadoop2-uber-2.7.5-1.8.0.jar.sha1
+        -
+          name: "Pre-bundled Hadoop 2.8.3"
+          category: "Pre-bundled Hadoop"
+          scala_dependent: false
+          id: 180-bundled-hadoop-283
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.8.3-1.8.0/flink-shaded-hadoop2-uber-2.8.3-1.8.0.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-shaded-hadoop2-uber/2.8.3-1.8.0/flink-shaded-hadoop2-uber-2.8.3-1.8.0.jar.asc
```

Comment:
Can not download, we got `404 - Path /org/apache/flink/flink-shaded-hadoop2-uber/2.8.3-1.8.0/flink-shaded-hadoop2-uber-2.8.3-1.8.0.jar not found in group repository "Public Repositories" [id=public].`